src/index.html

```diff
@@ -6,7 +6,71 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="container">
+      <div class="calendar calendar__month--length--29">
+        <div class="calendar__day day day--1 day--start--sun"></div>
+
+        <div class="calendar__day day day--2"></div>
```

Comment:
You shouldn't create new block (day) and modifier day--n. calendar__day as an element of calendar is ok.